platform_channels/android/app/src/main/kotlin/dev/flutter/platform_channels/AccelerometerStreamHandler.kt

```diff
@@ -29,9 +29,12 @@ class AccelerometerStreamHandler(sManager: SensorManager, s: Sensor) : EventChan
     override fun onAccuracyChanged(sensor: Sensor?, accuracy: Int) {}
 
     override fun onSensorChanged(sensorEvent: SensorEvent?) {
```

Comment:
I'm curious why the argument is a `SensorEvent?`. If we pare it down to a `SensorEvent` we can drop the null check from the body of this method.

Reply:
As far as I remember the argument was `SensorEvent?` for sending errors to Dart side. We do this for iOS, but I forgot to do the same on Android side.

Reply:
As long as it is intentional I'm fine. =)